src/redux/actions/instructorActions.js

```diff
@@ -1 +1,54 @@
 // Declare Actions
+import axios from 'axios';
+
+export const ERROR_ACTION = 'ERROR';
+export const GET_USER_ACTION = 'GET_USERS';
```

Comment:
Thanks for taking that suggestion, I think this will help make these actions more clear when we start to add more.  :)